pyrefinebio/config.py

```diff
@@ -9,12 +8,36 @@ class Config:
 
     _instance = None
 
-    def __new__(cls, config_file=None):
+    def __new__(cls):
         if not cls._instance:
             cls._instance = super(Config, cls).__new__(cls)
 
             cls.config_file = (
-                os.getenv("CONFIG_FILE") or config_file or str(Path.home()) + "/.refinebio.yaml"
+                os.getenv("CONFIG_FILE") or str(Path.home()) + "/.refinebio.yaml"
             )
 
+            config = {}
+
+            if os.path.exists(cls.config_file):
+                with open(cls.config_file) as config_file:
+                    config = yaml.full_load(config_file) or {}
+
+            cls.token = config.get("token") or ""
```

Comment:
`.get()` actually takes a second param that does that!
```suggestion
            cls.token = config.get("token", "")
```